benchmark/server/main.go

```diff
@@ -16,6 +16,15 @@
  *
  */
 
+/*
+Package main provides a server used for benchmarking.  It launches a server
+which is listening on port 50051.  An example to start the server can be found
+at:
+	benchmark/server/main.go
```

Comment:
This is the mentioned file. Please provide the example on how to run the server here.